src/entity-manager/EntityManager.ts

```diff
@@ -532,7 +532,7 @@ export class EntityManager {
         if (!ids.length)
             return Promise.resolve([]);
         const metadata = this.connection.getMetadata(entityClass);
-        const qb = this.createQueryBuilder(entityClass, FindOptionsUtils.extractFindManyOptionsAlias(optionsOrConditions) || metadata.name);
+        const qb = this.createQueryBuilder(entityClass, FindOptionsUtils.extractFindManyOptionsAlias(optionsOrConditions) || metadata.name, this.queryRunner);
```

Comment:
I think you don't need to pass `this.queryRunner` here since `this.createQueryBuilder` already uses it.